### PR TITLE
Updated README.md to help users to deploy team-edition using helm v3.

### PR DESCRIPTION
**Pull request by vijaynag-bs**

### CODE REVIEW

File: charts/mattermost-team-edition/README.md [hunk 1]
```diff
@@ -19,15 +20,15 @@ cluster using the [Helm](https://helm.sh) package manager.
...
+- Helm v3
+- [Migrate from Helm v2 to Helm v3](https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/)
 
 ## Installing the Chart
 
 To install the chart with the release name `my-release`:
 
 ```bash
-$ helm install --name my-release stable/mattermost-team-edition
+$ helm install my-release stable/mattermost-team-edition
```
Comment (cpanato) [inline charts/mattermost-team-edition/README.md:31]: this is only valid for helm3 not for helm2

maybe you can make a new section for helm3 and how to install using helm3 commands

Comment (vijaynag-bs) [reply charts/mattermost-team-edition/README.md:36]: Have used bold font for helm v3 commands. If it is fine, will extend to other charts. Thanks

**Comment by cpanato** [comment]
@vijaynag-bs can you udpate the branch and resolve the conflict then we can merge it. thanks!

File: charts/mattermost-team-edition/README.md [hunk 2]
```diff
@@ -85,7 +91,7 @@ Parameter                             | Description
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
 
 ```bash
-$ helm install --name my-release \
+$ helm install my-release \
```
Comment (cpanato) [inline charts/mattermost-team-edition/README.md:94]: @vijaynag-bs I'm sorry, but this is for helm 3 and not helm 2, can you keep this? same below

Comment (vijaynag-bs) [reply charts/mattermost-team-edition/README.md]: @cpanato Sorry for the mistake. Have changed

File: charts/mattermost-team-edition/README.md [hunk 3]
```diff
@@ -7,6 +7,7 @@
 ```bash
 $ helm repo add mattermost https://helm.mattermost.com
 $ helm install mattermost/mattermost-team-edition \
+  --generate-name \
```
Comment (cpanato) [inline charts/mattermost-team-edition/README.md:10]: @vijaynag-bs there is not `--generate-name` for helm 2

Comment (vijaynag-bs) [reply charts/mattermost-team-edition/README.md:10]: That was supposed to helm v3. Should I change to helm v2 ??

Comment (vijaynag-bs) [reply charts/mattermost-team-edition/README.md]: Sorry, I have changed. It was rectified in two lines.

**Comment by cpanato** [comment]
@vijaynag-bs can you update this PR then we can merge it?

**Comment by vijaynag-bs** [comment]
> @vijaynag-bs can you update this PR then we can merge it?

**Comment by vijaynag-bs** [comment]
@cpanato Updated. Thanks!

File: charts/mattermost-team-edition/Chart.yaml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Mattermost Team Edition server.
 name: mattermost-team-edition
-version: 3.13.1
-appVersion: 5.25.1
+version: 3.13.2
+appVersion: 5.25.0
```
Comment (cpanato) [inline charts/mattermost-team-edition/Chart.yaml:5]: sorry this should stay in 5.25.1

Comment (vijaynag-bs) [reply charts/mattermost-team-edition/Chart.yaml]: Sorry. Resolved now. Have kept chart version unchanged.

**Pull request merged**